### ISSUE TITLE
Need help with paleo related terms, filter and display

### ISSUE THREAD

**Issue by MortenHofft** [gbif/gbif-web#497]
I need help to figure out how these new terms are best represented on records and in search. Else I suepect I will look as amatures, when in fact it is just me not knowing this domain. 

https://paleobiodb.org/classic/displaySearchColls
http://paleoportal.org/index.php?globalnav=time_space&sectionnav=map
are the references that I've found. But it isn't obvious how to map them, nor if this is the ideal way to do it. 

earliestEonOrLowestEonothem
latestEonOrHighestEonothem
earliestEraOrLowestErathem
latestEraOrHighestErathem
earliestPeriodOrLowestSystem
latestPeriodOrHighestSystem
earliestEpochOrLowestSeries
latestEpochOrHighestSeries
earliestAgeOrLowestStage
latestAgeOrHighestStage
lowestBiostratigraphicZone
highestBiostratigraphicZone
group (reserved word _group)
formation
member
bed

**Comment by tkarim** [comment]
Some of us paleo folks chatted this morning and think one search for the 10 chronostratigraphy terms is ideal. It might change how we approach some of the vocabulary mapping, but it would be a great start.

**Comment by CecSve** [comment]
We could reduce the `geological context` search field to:

- `Chronostratigraphy` as one search field (based on vocabulary with concepts, ranks and ranges), called `Geological time`
- `Lithostratigraphy` for `group`, `formation`, `member` and `bed`
- `Biostratigraphy` for `lowest` and `highest biostratigraphy`

**Comment by MortenHofft** [comment]
For `Lithostratigraphy` and `Biostratigraphy`:

We should process them to normalise common abbreviations, misspellings and such - but there isn't a vocabulary as such.

And then it should be indexed in a way that allows wild card search (currently it is a keyword term).

@CecSve I assume it is a list of values . So for Lithostratigraphy, it would be `Lithostratigraphy: [bed, formation, group, member]`?

**Comment by CecSve** [comment]
> For `Lithostratigraphy` and `Biostratigraphy`:
> 
> We should process them to normalise common abbreviations, misspellings and such - but there isn't a vocabulary as such.
> 
> And then it should be indexed in a way that allows wild card search (currently it is a keyword term).
> 
> @CecSve I assume it is a list of values . So for Lithostratigraphy, it would be `"lithostratigraphy": [bed, formation, group, member]`?

Yes, I believe so.

**Comment by RogerBurkhalter** [comment]
I strongly agree with the concept of expanding searches for these terms. However, it would be a very tall order to produce or harvest a controlled vocabulary in that thousands, if not tens of thousands, of possible combinations are possible. Additionally, these terms change not just over political boundaries but vary from surface to subsurface usage (fossils also occur in subsurface cores). Terms such as "bed" can be formal or informal (local) use. The informal terms may be the most accurate and descriptive, i.e. "bone-bed", "12-inch layer", etc, and are terms of inconsistent usage. Most countries and states/provences have geologic surveys where formal lithostratigraphic terms are recorded; beds, especially local use terms, not so much.

**Comment by CecSve** [comment]
> I strongly agree with the concept of expanding searches for these terms. However, it would be a very tall order to produce or harvest a controlled vocabulary in that thousands, if not tens of thousands, of possible combinations are possible. Additionally, these terms change not just over political boundaries but vary from surface to subsurface usage (fossils also occur in subsurface cores). Terms such as "bed" can be formal or informal (local) use. The informal terms may be the most accurate and descriptive, i.e. "bone-bed", "12-inch layer", etc, and are terms of inconsistent usage. Most countries and states/provences have geologic surveys where formal lithostratigraphic terms are recorded; beds, especially local use terms, not so much.

Thanks for your input @RogerBurkhalter! GBIF does not plan to make vocabularies for any lithostratigraphy fields currently. The idea is to standardize the values when possible for the interpreted field on GBIF.org (mostly just changing upper/lower case values etc.) The plan is to have a controlled vocabulary for chronostratigraphy according to [ICS](https://stratigraphy.org/chart#latest-version) (excluding regional terms for now).
